Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -902,7 +900,7 @@ void GStreamerMediaEndpoint::doSetRemoteDescription(const RTCSessionDescription&
             return;
         processSDPMessage(&message, [this](unsigned, StringView mid, const auto* media) {
             const char* mediaType = gst_sdp_media_get_media(media);
-            m_mediaForMid.set(mid.toString(), g_str_equal(mediaType, "audio") ? RealtimeMediaSource::Type::Audio : RealtimeMediaSource::Type::Video);
+            m_mediaForMid.set(mid.toString(), equalIgnoringASCIICase(mediaType, "audio"_s) ? RealtimeMediaSource::Type::Audio : RealtimeMediaSource::Type::Video);
```

Comment:
Is it really okay to change those comparisons from case-sensitive to case-insensitive?